### ISSUE TITLE
Internet Explorer fails to launch with IEDriverServer v3.9.0, standalone selenium

### ISSUE THREAD

**Issue by joeyparrish** [SeleniumHQ/selenium#5738]
## Meta -
OS:  Windows 10
Selenium Version:  3.11.0
Browser:  Internet Explorer
Browser Version:  11

## Expected Behavior -
With standalone selenium hub & node on Windows 10, IE 11 should launch with the latest IEDriverServer.

## Actual Behavior -
The node fails with:

```
Started InternetExplorerDriver server (64-bit)
3.9.0.0
Listening on port 50390
Only local connections are allowed
19:19:39.406 WARN [RemoteSession$Factory.performHandshake] - Failed to connect to localhost/0:0:0:0:0:0:0:1:50390
java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:50390
```

## Steps to reproduce -
1. Download and extract [Selenium Hub and Nodes files.zip](https://github.com/SeleniumHQ/selenium/files/1885510/Selenium.Hub.and.Nodes.files.zip).  Contents:
    - Selenium/selenium-server-standalone-3.11.0.jar  (standalone selenium jar)
    - Selenium/IEDriverServer.exe  (IEDriverServer v3.9.0)
    - Selenium/hub.json  (hub config)
    - Selenium/node_ie.json  (node config)
    - Selenium/start_hub.bat  (starts the standalone server in hub mode)
    - Selenium/start_node.bat  (starts the standalone server in node mode)
    - Selenium/launch.js  (nodejs script to launch IE via webdriver)
2. Get nodejs if you don't have it.  (I'm using the current LTS, 8.11, with node in the PATH)
3. Double-click on "start_hub" and "start_node" to set up a simple selenium grid, with hub and node running in their own terminal windows.
4. Open another terminal. (Window+R, "cmd")
5. Change directory into the "Selenium" folder you extracted from the zip.
6. Run command "npm install selenium-webdriver" to install the required node module.
7. Run command "node launch.js" to launch IE
8. Observe the error in the output of the node.

## Notes -
If you replace IEDriverServer.exe with v3.8.0, everything works.  This issue is preventing us from upgrading to the latest IE driver.  We would like to make sure everything is upgraded before we file bugs for other IE-related issues we are having, in case those have already been fixed.

I'm suspicious of the IPv6 IP given on the console.  I can't compare this to the IP used by v3.8.0, because it only seems to print an IP when it fails to connect.  I attempted to work around this by disabling IPv6, but I think I only know how to do this on the ethernet adapter, not the loopback.  Disabling IPv6 on the ethernet adapter had no effect on this issue.

**Comment by barancev** [comment]
Please retest using [latest prebuilt IEDriverServer binary executable checked into project source tree](https://github.com/SeleniumHQ/selenium/raw/master/cpp/prebuilt/x64/Release/IEDriverServer.exe) instead of version 3.9.0.0

**Comment by joeyparrish** [comment]
This is working with the 3.11.1 build from here: http://selenium-release.storage.googleapis.com/3.11/IEDriverServer_x64_3.11.1.zip

I see now two places where I may have gone wrong:

1. This download page linked from the wiki shows folders in lexicographical order, so 3.9 is below 3.11 and it is easy to miss 3.11 completely: http://selenium-release.storage.googleapis.com/index.html
2. The SeleniumHQ download page, in the section "The Internet Explorer Driver Server
"), links explicitly to 3.9.0: https://www.seleniumhq.org/download/

Since the latest driver seems to work, can you update the downloads page on seleniumhq to point to it?  And is it possible to have selenium-release page sort in a numerically-sensitive or version-sensitive way?  (I noticed that there's no date metadata on the folders, so you can't sort them by date.)

Thanks!

**Comment by joeyparrish** [comment]
@Indu19Sahu, please see the resolution [above](https://github.com/SeleniumHQ/selenium/issues/5738#issuecomment-379522125): you need to upgrade your IE driver to 3.11.1: http://selenium-release.storage.googleapis.com/3.11/IEDriverServer_x64_3.11.1.zip

**Comment by barancev** [comment]
The download link updated.

**Comment by joeyparrish** [comment]
Thank you!

**Comment by marcelitamo** [comment]
I have a similar problem.
Selenium Version: 3.11.0
Browser: Internet Explorer
Browser Version: 11
IEDriver: 3.11.1

When I run the test with the local driver I got this error:

org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:13183

and when I run witha remote server running server-standalone-3.11.0.jar I got this

org.openqa.selenium.WebDriverException: Illegal key values seen in w3c capabilities: [ensureCleanSession, platform, version]

I have tried version 3.12 too without any success.

Could anybody guide me and help me, pls?
